src/App.tsx

```diff
@@ -1,9 +1,18 @@
 import React, { Component } from 'react';
 import { withRouter } from 'react-router-dom';
 import { Auth } from 'aws-amplify';
-import logo from './logo.svg';
+import { connect } from 'react-redux';
 import './App.css';
 import ScreensRoot from './screens/Root';
+import { hasAuthenticated } from './actions';
+
+const mapDispatchToProps = dispatch => {
```

Comment:
Referring https://redux.js.org/api/bindactioncreators to write more concise `mapDispatchToProps`.